deploy/role_binding.yaml

```diff
@@ -0,0 +1,11 @@
+kind: RoleBinding
```

Comment:
Does the manifest need to be updated?

Reply:
I don't believe so. the e2e test framework just expected different file names

Reply:
After running the tests it seems like there are some rbac definitions we're missing... we'll have to update the manifests as well. I'll include those changes once I can get them all in place.